scripts/bootstrap.sh

```diff
@@ -4,13 +4,17 @@
 
 # Argument parsing
 pipenv_install_args=()
+skip_nodeenv=false
 
 while :; do
   case $1 in
     --ci)
       # Prevent the Pipfile.lock from being updated automatically. Just use
       # the pinned dependencies.
       pipenv_install_args+=( "--ignore-pipfile" )
+      # And assume that node has already been set up, as the CI system can
+      # cache that.
+      skip_nodeenv=1
```

Comment:
Nitpicky, but if you set this to `true`, then below it can be handled as `$skip_nodeenv ||` without the string comparison.